# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Parent  095b4c5eb206263ed2f2b36994d04de23abaaaed
Misc: Reduce thread indentation from 16px to 10px.

diff --git a/mail/themes/shared/mail/threadPane.css b/mail/themes/shared/mail/threadPane.css
--- a/mail/themes/shared/mail/threadPane.css
+++ b/mail/themes/shared/mail/threadPane.css
@@ -393,9 +393,9 @@ tr[is="thread-row"] td > .thread-contain
   box-sizing: border-box;
 }
 
 tr[is="thread-row"] .subject-line {
-  margin-inline-start: calc(16px * var(--thread-level));
+  margin-inline-start: calc(10px * var(--thread-level));
   pointer-events: none;
   /* Line height px exception to avoid vertical cut off of characters. This
      should follow and match the density variation height of the row. */
   line-height: 22px;
@@ -415,9 +415,9 @@ tr[is="thread-row"][multiline="true"] td
 }
 
 /* Indent "alignment column" as well. */
 tr[is="thread-row"][multiline="true"] td[alignmentcolumn="true"] {
-  padding-left: calc(6px + 16px * var(--thread-level));
+  padding-left: calc(6px + 10px * var(--thread-level));
 }
 
 /* Remove centering of the td */
 tr[is="thread-row"][multiline="true"] td {
@@ -705,9 +705,9 @@ tr[data-properties~="junk"] :is(.tree-vi
   fill-opacity: 0.7;
 }
 
 tr[is="thread-row"][multiline="true"] .correspondentcol-column {
-  background-position-x: calc(4px + 16px * var(--thread-level));
+  background-position-x: calc(4px + 10px * var(--thread-level));
   background-position-y: top 5px;
 }
 :root[uidensity="compact"] tr[is="thread-row"][multiline="true"] .correspondentcol-column {
   background-position-y: top 1px;
@@ -716,9 +716,9 @@ tr[is="thread-row"][multiline="true"] .c
   background-position-y: top 7px;
 }
 
 tr[is="thread-row"][multiline="true"] td[alignmentcolumn="true"].correspondentcol-column {
-  padding-inline-start: calc(var(--tree-header-cell-padding) + 16px * var(--thread-level)) !important;
+  padding-inline-start: calc(var(--tree-header-cell-padding) + 10px * var(--thread-level)) !important;
 }
 
 .correspondentcol-column:dir(rtl) {
   background-position-x: right 1px;
